casanovo/casanovo.py

```diff
@@ -154,18 +168,25 @@ def sequence(
     to sequence peptides. If evaluate is set to True PEAK_PATH must be
     one or more annotated MGF file.
     """
-    output = setup_logging(output, verbosity)
-    config, model = setup_model(model, config, output, False)
+    output_path, output_root = _setup_output(
+        output_dir, output_root, force_overwrite, verbosity
+    )
+    utils.check_dir_file_exists(output_path, f"{output_root}.mztab")
+    config, model = setup_model(model, config, output_dir, output_root, False)
```

Comment:
**todo:** Shouldn't this use the resolved `output_path` instead of `output_dir`, which can be None?

Reply:
**todo:** I also that the docstring for `setup_model` is outdated (not shown here in the diff). Please update that as well.